src/app/cheat-sheets/game-base/balancers/balancers.component.html

```diff
@@ -45,24 +34,21 @@
     <div class="col-12 col-md-5">
       <strong class="d-block text-center">Tips</strong>
       <ul>
-        <li>Don't overuse balancers, use them when needed.</li>
-        <li>Make a build that's inherently balanced, by letting the assembly machines consume a full belt of items.</li>
+        <li>Only use balancers when needed.</li>
+        <li>If your assembly machines consume a full belt you don't need a balancer.</li>
```

Comment:
this is the more importaint part:
"Make a build that's inherently balanced" wich can have multi scale meanings:
* A "small" belt build maybe using simply a proper ratio of machines like 3 coper wire to 2 green circuits
* A "medium" belt build where its 2 input belts of iron with 1 output belt of gears
* A  train build using a proper ratio of machines that take in 1:1 train of iron train and copper train

Having a balanced build does not guarantee on not needing balancers, but will alleaviate overuse